CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ The CKEditor 5 Collaboration features changelog can be found here: https://ckedi
 
 ### MAJOR BREAKING CHANGES [ℹ️](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/versioning-policy.html#major-and-minor-breaking-changes)
 
-* **[image](https://www.npmjs.com/package/@ckeditor/ckeditor5-image)**: In order to use the "insert image via URL" feature you now need to load the `ImageInsert` plugin and use the `imageInsert` button instead of the `imageUpload` button that implemented this functionality previously.
+* **[image](https://www.npmjs.com/package/@ckeditor/ckeditor5-image)**: In order to use the "insert image via URL" feature you now need to load the `ImageInsert` plugin and use the `insertImage` button instead of the `uploadImage` button that implemented this functionality previously.
```

Comment:
I thought it was needed, according to [this comment](https://github.com/ckeditor/ckeditor5/issues/8033#issuecomment-717763485):
> Update changed references in the changelog.

But I agree that altering the history is a bad thing 😉

Reply:
Hmm, that's weird. @mlewand do you remember what that one was about? I've missed that. But _maybe_ it was about writing a proper merge message so it would be documented in the future changelog.